# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/Send-binary-arraybufferview-int8.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/websockets/Send-binary-arraybufferview-int8.any.js.ini
--- a/testing/web-platform/meta/websockets/Send-binary-arraybufferview-int8.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/websockets/Send-binary-arraybufferview-int8.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -4,18 +4,15 @@
 
 [Send-binary-arraybufferview-int8.any.html?wpt_flags=h2]
   expected:
-    if (os == "linux") and debug and not swgl and not webrender: ["OK", "TIMEOUT"]
-    if (os == "linux") and debug and not swgl and webrender: TIMEOUT
-    if (os == "linux") and debug and swgl: ["OK", "ERROR", "TIMEOUT"]
+    if not swgl and (os == "linux") and debug and not webrender: ["OK", "TIMEOUT"]
+    if swgl and (os == "linux"): ["OK", "ERROR", "TIMEOUT"]
   [Send binary data on a WebSocket - ArrayBufferView - Int8Array - Connection should be opened]
     expected:
-      if (os == "linux") and debug and webrender: ["PASS", "NOTRUN"]
-      if (os == "linux") and debug and not webrender: ["PASS", "NOTRUN"]
+      if (os == "linux") and debug: ["PASS", "NOTRUN"]
 
   [Send binary data on a WebSocket - ArrayBufferView - Int8Array - Message should be received]
     expected:
-      if (os == "linux") and debug and webrender: ["PASS", "NOTRUN"]
-      if (os == "linux") and debug and not webrender: ["PASS", "NOTRUN"]
+      if (os == "linux") and debug: ["PASS", "NOTRUN"]
 
   [Send binary data on a WebSocket - ArrayBufferView - Int8Array - Connection should be closed]
     expected:
@@ -29,5 +26,4 @@
 
 [Send-binary-arraybufferview-int8.any.worker.html?wpt_flags=h2]
   expected:
-    if (os == "linux") and debug and not swgl and webrender: TIMEOUT
-    if (os == "linux") and debug and not swgl and not webrender: ["OK", "TIMEOUT"]
+    if (os == "linux") and not webrender and debug: ["OK", "TIMEOUT"]